src/sentry/api/endpoints/dif_files.py

```diff
@@ -7,30 +7,31 @@
 
 from sentry.utils import json
 from sentry.api.serializers import serialize
-from sentry.api.bases.chunk import ChunkAssembleMixin
 from sentry.api.bases.project import ProjectEndpoint, ProjectReleasePermission
-from sentry.models import File, ChunkFileState, ProjectDSymFile
-
-
-class DifAssembleEndpoint(ChunkAssembleMixin, ProjectEndpoint):
+from sentry.models import ChunkFileState, ProjectDSymFile, FileBlob, \
+    FileBlobOwner, get_assemble_status, set_assemble_status
+
+
+def find_missing_chunks(organization, chunks):
+    """Returns a list of chunks which are missing for an org."""
+    if not chunks:
+        return []
+    blobs = list(FileBlob.objects.filter(checksum__in=chunks))
```

Comment:
nit: here, you also only just need the ids, so you can `values_list` it as well.